README.md

```diff
@@ -51,6 +53,15 @@ and cleanup them with:
 ```bash
 envsubst < manifests/storage_checkup.yaml|kubectl delete -f -
 ```
+
+### SkipTeardown Modes
+
+The `skipTeardown` field provides more flexibility for debugging by allowing control over when teardown steps are skipped. The available modes are:
+
+- **SkipTeardownAlways (`always` or `true`)**: Skips the teardown process in all cases. This is useful when you want to keep resources for further inspection.
+- **SkipTeardownOnFailure (`onfailure`)**: Skips teardown only if a failure occurs. This is particularly helpful when debugging issues after a failure.
+- **SkipTeardownNever (`never` or `false`)**: Always performs the teardown steps, ensuring that all resources are cleaned up after the checkup run. This is the default behavior.
```

Comment:
Not sure users are interested in the const names. Maybe just something like:
- `always`/`true` - skips...

Reply:
ack